### PR TITLE
feature(ui-ux): added wallet accordion, password and recovery word FAQ screen

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
This PR contains
- Wallet accordion redesign
- Passcode FAQ Screen 
- Recovery FAQ Screen 

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/WalletNavigator/WalletNavigator.tsx [hunk 1]
```diff
@@ -266,6 +268,25 @@ export function WalletNavigator (): JSX.Element {
             headerTitle: translate('screens/WalletNavigator', 'Guidelines')
           }}
         />
+
+        <WalletStack.Screen
```
Comment (kyleleow) [inline mobile-app/app/screens/WalletNavigator/WalletNavigator.tsx:272]:
```suggestion
        <WalletStackV2.Screen
```

File: mobile-app/app/screens/WalletNavigator/WalletNavigator.tsx [hunk 2]
```diff
@@ -266,6 +268,25 @@ export function WalletNavigator (): JSX.Element {
...
+          component={RecoveryWordsFaqV2}
+          name='RecoveryWordsFaq'
+          options={{
+            headerTitle: translate('screens/WalletNavigator', 'About Recovery Words'),
+            headerRight: () => <></>
+          }}
+        />
+
+        <WalletStack.Screen
```
Comment (kyleleow) [inline mobile-app/app/screens/WalletNavigator/WalletNavigator.tsx:281]:
```suggestion
        <WalletStackV2.Screen
```

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/PasscodeFaqV2.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  return (
+    <ThemedScrollViewV2
+      contentContainerStyle={tailwind('pt-8 px-5 pb-16')}
+      style={tailwind('flex-1')}
+      testID='passcode_faq'
+    >
+      <ThemedTextV2
+        style={tailwind('text-base font-normal-v2 px-5')}
+      >
+        {translate('components/PasscodeFaq', 'Your six-digit numeric code passcode, set by you, would be used to authorize all transactions. Keep your passcode private and secure. You are responsible for your own security.')}
```
Comment (thedoublejay) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/PasscodeFaqV2.tsx:40]: Maybe better to change to `Your six-digit numeric passcode`

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/PasscodeFaqV2.tsx (hunk 3 above)
Comment (thedoublejay) [reply mobile-app/app/screens/WalletNavigator/screens/CreateWallet/PasscodeFaqV2.tsx:40]: Let's resolve this on another PR @fullstackninja864

**Pull request merged**